tests/test_pps.py

```diff
@@ -45,6 +45,18 @@ def test_list_job():
     assert len(jobs) >= 1
 
 
+def test_list_job_set():
+    sandbox = Sandbox("list_job_set1")
+    sandbox.wait()
+    client = sandbox.client
+
+    commit, _, _ = util.create_test_pipeline(client, "list_job_set2")
+    client.wait_commit(commit.id)
+
+    jobs = list(client.list_job_set())
+    assert len(jobs) == 4
```

Comment:
Why 4 instead of 2?

Reply:
Seems like in the scenario where an input and output repo are created, followed by a pipeline, there's an initial empty commit that propagates through. The user commit then becomes the second commit. https://pachyderm.slack.com/archives/C01M3RQNVCP/p1626725668243300